astroid/tests/unittest_nodes.py

```diff
@@ -464,6 +464,23 @@ def hello(False):
             self.assertEqual(del_true.name, "True")
 
 
+class AnnAssignNodeTest(unittest.TestCase):
+    def test_ann_assign(self):
```

Comment:
Please extend test suite for more complex cases found in [PEP526](https://www.python.org/dev/peps/pep-0526/):

```
primes: List[int] = []

captain: str  # Note: no initial value!

class Starship:
    stats: ClassVar[Dict[str, int]] = {}
```